homeassistant/components/cisco_webex_teams/notify.py

```diff
@@ -0,0 +1,52 @@
+"""Cisco Webex Teams notify component."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    PLATFORM_SCHEMA, BaseNotificationService, ATTR_TITLE)
+from homeassistant.const import (CONF_TOKEN)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['webexteamssdk==1.1.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ROOMID = 'roomid'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_TOKEN): cv.string,
+    vol.Required(CONF_ROOMID): cv.string,
+})
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the CiscoWebexTeams notification service."""
+    return CiscoWebexTeamsNotificationService(
+        config[CONF_TOKEN],
+        config[CONF_ROOMID])
+
+
+class CiscoWebexTeamsNotificationService(BaseNotificationService):
+    """The Cisco Webex Teams Notification Service."""
+
+    def __init__(self, token, room):
+        """Initialize the service."""
+        from webexteamssdk import WebexTeamsAPI
+        self.room = room
+        self.client = WebexTeamsAPI(access_token=token)
```

Comment:
Can we check that the token is ok before finishing platform set up?

Reply:
I don't believe there is any way to check that initially without an action such as sending an actual message.

Reply:
Sounds good!